src/components/navigation/UserMenu.vue

```diff
@@ -5,7 +5,6 @@
     </a>
     <vda-account
       :logo="logo"
-      :openUrl="openUrl"
```

Comment:
I disagree with removing the openURL.
The bug is on the client side, not on this demo

Reply:
Noted, 

Actually, all issues resolve around validation and minor text typos and corrections.

will create different PRs for each issue moving forward.

Reply:
@aurelticot, 
I have added the `OpenUrl` prop back